src/System.Drawing.Common/tests/IconTests.cs

```diff
@@ -0,0 +1,586 @@
+﻿using System.Collections.Generic;
```

Comment:
@hughbe as you probably know if we copy code from Mono we have to mess with the copyright header as below (also we need one :))

```
	1. Keep the existing copyright headers in place
		○ Note: If you are porting just portions of the file over, you still need to bring over all the existing copyright headers.
	2. Add the following copyright headers – NOTE this is missing the middle line of the ‘regular’ CoreFx copyright headers:
		// Licensed to the .NET Foundation under one or more agreements.
                // See the LICENSE file in the project root for more information.
```

If it's bits of Mono or inspired by bits of Mono - well I'm not a lawyer but it seems to me we should do the same thing.